test-requirements.txt

```diff
@@ -10,5 +10,5 @@ pytest-xdist==1.26.0
 shade==1.22.2
 twine
 wheel==0.30.0
-yapf==0.21.0
+yapf>=0.25.0,<2
```

Comment:
It seems like it's not needed at all in this file now.

Reply:
Other than that the PR is good to merge

Reply:
Tracking this as https://github.com/ansible/molecule/issues/1710

Reply:
I'd like to have one way of doing things + adding useless requirements to envs where they aren't used is increasing setup step